L1Trigger/CSCTriggerPrimitives/src/CSCGEMMotherboard.cc

```diff
@@ -15,6 +14,7 @@ CSCGEMMotherboard::CSCGEMMotherboard(unsigned endcap, unsigned station,
 {
   // super chamber has layer=0!
   gemId = GEMDetId(theRegion, 1, theStation, 0, theChamber, 0).rawId();
+  std::cout <<"CSCGEMMotherboard GEMDetID "<< gemId <<" CSC station "<< theStation <<" ring "<< theRing << std::endl;
```

Comment:
This line is filling hundreds of lines in the step2 logs (in a thread unsafe way, moreover!)
Removed in #24898

Reply:
@perrotta I just had @dildick submit #24896 to fix this